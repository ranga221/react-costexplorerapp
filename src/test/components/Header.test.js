import React from 'react';
import Header from '../../components/Header';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import {shallow} from 'enzyme';


test('testing Header', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();

    
    //expect(wrapper.find('h1').text()).toBe('Cost Explorer');
//const renderer = new ReactShallowRenderer();
//renderer.render(<Header />);
//expect(renderer.getRenderOutput()).toMatchSnapshot();
});