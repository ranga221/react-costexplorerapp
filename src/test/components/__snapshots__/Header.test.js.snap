// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Header 1`] = `
<header>
  <h1>
    Cost Explorer
  </h1>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
     Home 
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/create"
  >
     Create New
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/edit"
  >
     Edit Existing
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/help"
  >
     Help Page
  </NavLink>
</header>
`;
